generators/app/templates/travis.yml

```diff
@@ -1,16 +1,19 @@
 sudo: false
 language: node_js
 cache:
+  yarn: true
   directories:
     - node_modules
 notifications:
   email: false
 node_js:
   - '6'
+before_script:
+  - yarn global add codecov
 script:
   - yarn start validate
 after_success:
-  - yarn start report-coverage
+  - codecov
   - yarn start release
```

Comment:
I already did the `semantic-release` stuff to the generator here, but I look forward to the PRs on the other projects. Thanks!